x-pack/dockerlogbeat/docs/configuration.asciidoc

```diff
@@ -61,242 +58,62 @@ format is `"username:password"`.
-load balancing mode (`output.elasticsearch.loadbalance`) set to `true`. Example:
-If you have 2 hosts and 3 workers, in total 6 workers are started (3 for each
-host).
-
-|=====
-
-
-[float]
-[[ls-output-options]]
-=== {ls} output options
-
-[options="header"]
-|=====
-|Option  | Default | Description
-
-|`output.logstash.hosts`
-|`"localhost:5044"` 
-|The list of known {ls} servers to connect to. If load balancing is
-disabled, but multiple hosts are configured, one host is selected randomly
-(there is no precedence). If one host becomes unreachable, another one is
-selected randomly. If no port is specified, the default is `5044`.
-
-|`output.logstash.index`
-|
-|The index root name to write events to. For example +"dockerlogs"+ generates
-+"dockerlogs-{version}"+ indices. 
-
-3+|*Advanced:*
-
-|`output.logstash.backoff.init`
-|`1s`
-|The number of seconds to wait before trying to reconnect to {ls} after
-a network error. After waiting `backoff.init` seconds, the {log-driver}
-tries to reconnect. If the attempt fails, the backoff timer is increased
-exponentially up to `backoff.max`. After a successful connection, the backoff
-timer is reset.
-
-|`output.logstash.backoff.max`
-|`60s`
-|The maximum number of seconds to wait before attempting to connect to
-{ls} after a network error.
-
-|`output.logstash.bulk_max_size`
-|`2048`
-|The maximum number of events to bulk in a single {ls} request. Specify 0 to
-allow the queue to determine the batch size. 
-
-|`output.logstash.compression_level`
-|`0`
-|The gzip compression level. Valid compression levels range from 1 (best speed)
-to 9 (best compression). Specify 0 to disable compression.  Higher compression
-levels reduce network usage, but increase CPU usage.
-
-|`output.logstash.escape_html`
-|`false`
-|Whether to escape HTML in strings.
+|The http request timeout in seconds for the Elasticsearch request.
 
-|`output.logstash.loadbalance`
-|`false`
-|Whether to load balance when sending events to multiple {ls} hosts. If set to
-`false`, the driver sends all events to only one host (determined at random) and
-switches to another host if the selected one becomes unresponsive.
-
-|`output.logstash.pipelining`
-|`2`
-|The number of batches to send asynchronously to {ls} while waiting for an ACK
-from {ls}. Specify 0 to disable pipelining.
-
-|`output.logstash.proxy_url`
+|`proxy_url`
 |
-|The URL of the SOCKS5 proxy to use when connecting to the {ls} servers. The
-value must be a URL with a scheme of `socks5://`. You can embed a
-username and password in the URL (for example,
-`socks5://user:password@socks5-proxy:2233`).
-
-|`output.logstash.proxy_use_local_resolver`
-|`false`
-|Whether to resolve {ls} hostnames locally when using a proxy. If `false`,
-name resolution occurs on the proxy server.
+|The URL of the proxy to use when connecting to the Elasticsearch servers. The
+value may be either a complete URL or a "host[:port]", in which case the "http"
```

Comment:
```suggestion
value may be either a complete URL or a `host[:port]`, in which case the `http`
```

Reply:
Let's wrap values in backticks. If you want to show the quotation marks, you can add them back in (I wasn't sure).